docs/settingUp.md

```diff
@@ -23,6 +23,14 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
+  
+ **Verification** : Use this command `git remote -v` and the following lines should be part of the output:
+
+  `upstream        https://github.com/TEAMMATES/teammates.git (fetch)`
+
+  `upstream        https://github.com/TEAMMATES/teammates.git (push)`
```

Comment:
Consider the following instead:
```sh
upstream	https://github.com/TEAMMATES/teammates.git (fetch)
upstream	https://github.com/TEAMMATES/teammates.git (push)
```

If you are wondering, what you have right now will be displayed as:
`upstream        https://github.com/TEAMMATES/teammates.git (fetch)`
`upstream        https://github.com/TEAMMATES/teammates.git (push)`